build_sdk.yml

```diff
@@ -10,10 +10,14 @@ on:
 
 env:
   OAS_PATH: { YOUR_SPEC_PATH }
-  CONFIG_PATH: { YOUR_CONFIG_PATH } # optional
+  # Optional.
+  CONFIG_PATH: { YOUR_CONFIG_PATH }
   STAINLESS_ORG_NAME: { YOUR_ORG_NAME }
   STAINLESS_PROJECT_NAME: { YOUR_PROJECT_NAME }
-  FAIL_RUN_ON: "error" # "note" | "warning" | "error" | "fatal" | "never"
+  # "never" | "note" | "warning" | "error" | "fatal"
+  FAIL_RUN_ON: "error"
+  # Optional. Only used for commenting on PRs.
+  GITHUB_TOKEN: ${{ github.token }}
 
 jobs:
   preview-sdk:
```

Comment:
More nitting: I don't like the aliteration with `Build SDK / preview-sdk`, wish we could call this `Stainless / Preview changes` or something

Reply:
we can just call this `preview` :shrug: